src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -286,6 +286,8 @@ namespace OpenMS
     */
     void sortByPosition();
 
+    void sortByPositionPresorted(const std::vector<std::pair<Size, bool>>& chunks);
```

Comment:
use doxygen to describe what `chunks` is

Reply:
Gonna do that!